onnxruntime/python/__init__.py

```diff
@@ -2,3 +2,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
+
+from .onnxruntime_cuda_temp_env import load_nvidia_libs
+
+load_nvidia_libs()
```

Comment:
The __init__.py is also used by cpu only package. We do not need load cuda dlls for cpu.

Reply:
Added `if __package__ == "onnxruntime-gpu"`

Reply:
Good. Please see the updated comment.